Mathlib/LinearAlgebra/BilinearForm/TensorProduct.lean

```diff
@@ -59,36 +59,36 @@ def tensorDistrib : BilinForm A M₁ ⊗[R] BilinForm R M₂ →ₗ[A] BilinForm
 theorem tensorDistrib_tmul (B₁ : BilinForm A M₁) (B₂ : BilinForm R M₂) (m₁ : M₁) (m₂ : M₂)
     (m₁' : M₁) (m₂' : M₂) :
     tensorDistrib R A (B₁ ⊗ₜ B₂) (m₁ ⊗ₜ m₂) (m₁' ⊗ₜ m₂')
-      = B₂ m₂ m₂' • B₁ m₁ m₁' :=
-  rfl
```

Comment:
If something can be proved with `rfl`, it is usually best not to try and prove it with a different lemma; only `rfl` makes this eligible for `dsimp`.

Reply:
Put back to rfl.